scripts/initialize.sh

```diff
@@ -79,24 +79,24 @@
 set -ex
 $PROV_CMD init testing --custom-denom "$DENOM" $arg_timeout_commit $arg_chain_id
 $PROV_CMD keys add validator $arg_keyring
-$PROV_CMD add-genesis-root-name validator pio $arg_keyring
-$PROV_CMD add-genesis-root-name validator pb --restrict=false $arg_keyring
-$PROV_CMD add-genesis-root-name validator io --restrict $arg_keyring
-$PROV_CMD add-genesis-root-name validator provenance $arg_keyring
-$PROV_CMD add-genesis-account validator "100000000000000000000$DENOM" $arg_keyring
-$PROV_CMD gentx validator "1000000000000000$DENOM" $arg_chain_id $arg_keyring
-$PROV_CMD add-genesis-marker "100000000000000000000$DENOM" \
+$PROV_CMD genesis add-root-name validator pio $arg_keyring
+$PROV_CMD genesis add-root-name validator pb --restrict=false $arg_keyring
+$PROV_CMD genesis add-root-name validator io --restrict $arg_keyring
+$PROV_CMD genesis add-root-name validator provenance $arg_keyring
+$PROV_CMD genesis add-account validator "100000000000000000000$DENOM" $arg_keyring
```

Comment:
We'll fix this if it becomes a problem.

Reply:
Some of these variables specifically need to not be quoted because they need to be expanded by the command, or are sometimes empty, but need to not be an empty string arg.